com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -36,7 +36,8 @@
     public static final String APPLICATION_ASSOCIATED_WITH_ANOTHER_MTA = "Application \"{0}\" exists and is associated with MTA \"{1}\"";
     public static final String APPLICATION_EXISTS_AS_STANDALONE = "Application \"{0}\" exists, but is not associated with any MTA";
     public static final String SERVICE_ASSOCIATED_WITH_OTHER_APPS = "Service \"{0}\" already exists, but is bound to application(s), not detected as part of the mta - standalone apps : {1}";
-    public static final String SERVICE_ASSOCIATED_WITH_OTHER_MTAS = "Service \"{0}\" already exists, but is associated with MTA(s): {1}";
+    public static final String SERVICE_ASSOCIATED_WITH_OTHER_MTAS = "Service \"{0}\" already exists, but is associated with MTA(s) that have not listed it as shared: {1}";
+    public static final String SERVICE_SHOULD_BE_SHARED = "Service \"{0}\" can not be deployed as not shared because it is already set to shared by MTA(s): {1}";

```

Comment:
You can not use service {1}, because the following MTA(s) do not declare it as shared: {2} 
You have to declare service {1} as shared, because it is already shared with MTA(s): {2}

Reply:
Service {1} can not be used, because the following MTA(s) do not declare it as shared: {2}
Service {1} has to be declared as shared, because it is already shared with MTA(s): {2}